module/userProjects/helper.js

```diff
@@ -11,6 +11,7 @@ const coreService = require(GENERICS_FILES_PATH + "/services/core");
 const libraryCategoriesHelper = require(MODULES_BASE_PATH + "/library/categories/helper");
 const projectTemplatesHelper = require(MODULES_BASE_PATH + "/project/templates/helper");
 const projectTemplateTasksHelper = require(MODULES_BASE_PATH + "/project/templateTasks/helper");
+const { util } = require('chai');
```

Comment:
@priyanka-TL where do we use chai package? Also where is the code to set completedDate